hack/makefile-vars.mk

```diff
@@ -0,0 +1,25 @@
+# Build options
+IMG_BUILD_CMD ?= docker
+TARGET_OS ?= linux
+TARGET_ARCH ?= amd64
+
+# Tools versions
+OPM_VERSION ?= v1.26.0
+
+# Tools paths
+OUTDIR ?= build/_output
+OUTDIR_TOOLS= ${OUTDIR}/tools
+OPM ?= ${OUTDIR_TOOLS}/opm-${OPM_VERSION}
+
+# Image name vars
+IMG_REGISTRY ?= quay.io
+REGISTRY_NAMESPACE ?= ocs-dev
+IMG_TAG ?= latest
+
+OCS_BUNDLE_IMG_NAME ?= ocs-operator-bundle
+OCS_CATALOG_IMG_NAME ?= ocs-operator-catalog
+
+OCS_BUNDLE_IMG ?= ${IMG_REGISTRY}/${REGISTRY_NAMESPACE}/${OCS_BUNDLE_IMG_NAME}:${IMG_TAG}
+OCS_CATALOG_IMG ?= ${IMG_REGISTRY}/${REGISTRY_NAMESPACE}/${OCS_CATALOG_IMG_NAME}:${IMG_TAG}
+
+NOOBAA_BUNDLE_IMG ?= quay.io/noobaa/noobaa-operator-bundle:v5.12.0
```

Comment:
If you want these in makefile, remove it from scripts. Let's not have duplicates.

Reply:
As discussed earlier, We have decided to not remove anyhting from the existing scripts as they will cause a chain reaction as these scripts are used by DS build team and also CI environment. As we are in the middle of the refactor we have to keep these as a new copy for the new make targets as I add them 1 by 1. Once we are clear with all our new make targets, We will remove the common.sh vars after informing the DS build team & also making changes in the CI env.

Reply:
> We have decided to not remove anyhting from the existing scripts as they will cause a chain reaction as these scripts are used by DS build team and also CI environment.
>
This is exactly why we need to be more thoughtful about the changes we make. Do we really have no other option than making a breaking change?